app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+                  > ${recentProjects[4].workStack[2]}</a
+                >
+              </li>
+            </ul>`;
+    } else if (i === 5) {
+      desktopProjectDiv.classList.add('project-3', 'project-block');
+      desktopProjectDiv.innerHTML = `<h3 class="project-title">${recentProjects[5].name}</h3>
+            <p class="project-details">
+             ${recentProjects[5].description}
+            </p>
+            <ul class="project-technology">
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="project-btn project-btn-1 active-btn-effect hover-btn-effect"
+                >
+                  ${recentProjects[5].workStack[0]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://getbootstrap.com/"
+                  class="project-btn project-btn-2 active-btn-effect hover-btn-effect"
+                  >${recentProjects[5].workStack[1]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="project-btn project-btn-3 active-btn-effect hover-btn-effect"
+                  >${recentProjects[5].workStack[2]}</a
+                >
+              </li>
+            </ul>
+          `;
+    }
+
+    if (desktopProjectDiv.innerHTML !== '') {
+      descktopProjectsWrapperDiv.appendChild(desktopProjectDiv);
+    }
+  }
+}
+
+// portfolio end
+
+// popup start
+
+const modal = document.querySelector('.project-details-popup');
+const overlay = document.querySelector('.overlay');
+const openModalBtn = document.querySelectorAll('.pop');
+
+overlay.classList.add('hidden');
+const popupContainer = document.getElementById('project-details-popup');
+popupContainer.innerHTML = ` <div class="project-header">
+        <h3 class="project-title-popup">Multi-Post Stories</h3>
+        <i class="fa fa-close popup-close" id="close-x"></i>
+      </div>
+      <div>
+        <ul class="project-technology-popup">
+          <li>
+            <a
+              href="https://html.com/"
+              class="project-btn-pop project-btn-1-pop active-btn-effect hover-btn-effect"
+            >
+              HTML</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://getbootstrap.com/"
+              class="project-btn-pop project-btn-2 active-btn-effect hover-btn-effect"
+              >Bootstrap</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://www.ruby-lang.org/en/"
+              class="project-btn-pop project-btn-3 active-btn-effect hover-btn-effect"
+              >Ruby on Rails</a
+            >
+          </li>
+        </ul>
+      </div>
+      <div class="project-more">
+        <div>
+          <img
+            src="images/Snapshoot Portfolio.png"
+            alt=""
+            class="popup-project-img"
+          />
```

Comment:
- [ ] 👋 Thank you for your dedication to the project! I noticed that the popup image in the desktop version is not aligned properly, as it should be 60% of the width of the modal, according to the design requirements. To address this, I recommend updating your code to ensure that the popup image is properly aligned and occupies 60% of the modal's width in the desktop version.


![image](https://github.com/FilbertKimasa/Portfolio/assets/55840999/2cd537f8-0b5e-4b53-9117-6f11d1559f56)


By making this adjustment, your project will adhere more closely to the design, providing a visually appealing and consistent user experience. Keep up the good work! 🌟💻🚀